bin/binaryaudit

```diff
@@ -18,6 +18,12 @@ args = cli.arg_parser.parse_args()
 
 util.set_verbosity(args.verbose)
 
+use_suppressions = not args.no_suppressions
```

Comment:
Seems this one is not necessary and is not reflected in the arg parser anymore.

Thanks